_episodes/03-computing-pi.md

```diff
@@ -133,7 +146,7 @@ Numba makes it easier to create accellerated functions. You can use it with the
 import numba
 
 @numba.jit
-def sum_range(a: int):
+def numbified_sum_range(a: int):
```

Comment:
maybe `numba_sum_range` is a better name

Reply:
Agree.